contracts/rewards/src/contract.rs

```diff
@@ -191,8 +191,8 @@ mod tests {
     fn test_rewards_flow() {
         let chain_name: ChainName = "mock-chain".parse().unwrap();
         let user = Addr::unchecked("user");
-        let worker = Addr::unchecked("worker");
-        let worker_contract = Addr::unchecked("worker contract");
+        let verifier = Addr::unchecked("verifier");
+        let participation_contract = Addr::unchecked("participation contract");
```

Comment:
what is a participation contract?

Reply:
it's the contract associated with the rewards pool, so like voting verifier or multisig prover, it was called worker_contract and I don't think calling it verifier_contract is fitting